components/Banner.tsx

```diff
@@ -2,19 +2,27 @@ import Image from "next/image";
 import React, { useEffect, useState } from "react";
 import Button from '../components/Button';
 import chromeLogo from '../public/chrome-logo.png'
+import { useSession } from "next-auth/react";
+import type { Session } from "next-auth";
+import RudderContext from "./RudderContext";
+import { getAndSetAnonymousIdFromLocalStorage } from "../utils/rudderstack_initialize";
+import { getAuthUserId, getAuthUserName } from "../utils/auth";
 
 export type BannerHeightType = "h-12" | "h-24" | "h-32" | "h-40" | "h-44" | undefined;
-type BannerSituation = "not-installed" | "incompatible-browser" | "incompatible-device" | null;
+type BannerSituation = "extension-not-installed" | "incompatible-browser" | "incompatible-device" | null;
```

Comment:
Please don't change these types since the event name has been decided based on these types only.